l2lchannel.js

```diff
@@ -0,0 +1,128 @@
+import L2LClient from "lively.2lively/client.js";
+import { string, num, promise, fun } from "lively.lang";
+
+export class Channel {
+  constructor(senderRecvrA, onReceivedMethodA, senderRecvrB, onReceivedMethodB){
+      if (!senderRecvrA) throw new Error("no sender / receiver a!");
+      if (!senderRecvrB) throw new Error("no sender / receiver b!");
+      if (typeof senderRecvrA[onReceivedMethodA] !== "function") throw new Error(`sender a has no receive method ${onReceivedMethodA}!`);
+      if (typeof senderRecvrB[onReceivedMethodB] !== "function") throw new Error(`sender b has no receive method ${onReceivedMethodB}!`);           
+      this.senderRecvrA = senderRecvrA;
+      this.senderRecvrA.l2lclient ? {} : this.senderRecvrA.l2lclient = Channel.makeL2LClient()
```

Comment:
What is this structure? Why not simply an if instead of a tertiary with an emtpy object?